pkg/client/config.go

```diff
@@ -76,6 +81,24 @@ func SaveConfig(config map[string]string) error {
 	return json.NewEncoder(configFile).Encode(&config)
 }
 
+func (c VeleroConfig) Namespace() string {
+	ns, ok := c[ConfigKeyNamespace]
+	if !ok {
```

Comment:
Very subjective, but I think this and the next function would read better if the check on the map key was all inlined.

Reply:
I think we can't do this here since `val` is used outside the `if...` block?